backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Mod.java

```diff
@@ -123,4 +132,15 @@ public boolean usesQuotedText() {
         NodeParser.getNodeFromExpression("./quotedText", this.node);
     return newContentNode.isPresent();
   }
+
+  /**
+   * Checks weather a quotedStructure was used for a substitution.
```

Comment:
Typo, should be "whether" not "weather" ;) Also in the java docs of method `usesQuotedText`